### PR TITLE
RPP Test suite - Display all cases for "--help" request

### PR DESCRIPTION
**Pull request by r-abishek**
This PR adds a "Functionality Reference List" display to make it easier for the user and display case numbers with corresponding augmentation names when the user asks for "--help" in any of the 4 test suite scripts in RPP (image/voxel/audio/misc).

### CODE REVIEW

**Comment by r-abishek** [comment]
Sample output:
```
python3 runTests.py --help
```
image

**Review by rrawther — CHANGES_REQUESTED** [review]
Please address the review comments

File: utilities/test_suite/HIP/runMiscTests.py [hunk 1]
```diff
@@ -121,6 +121,10 @@ def rpp_test_suite_parser_and_validator():
     parser.add_argument('--qa_mode', type = int, default = 0, help = "Run with qa_mode? Outputs from tests will be compared with golden outputs - (0 / 1)", required = False)
     parser.add_argument('--batch_size', type = int, default = 1, help = "Specifies the batch size to use for running tests. Default is 1.")
     parser.add_argument('--preserve_output', type = int, default = 1, help = "preserves the output of the program - (0 = override output / 1 = preserve output )" )
+    if '--help' or '-h' in sys.argv:
```
[the remark this answers is not in the document]
Comment (r-abishek) [reply utilities/test_suite/HIP/runMiscTests.py]: addressed

**Comment by r-abishek** [comment]
@rrawther @kiritigowda Changes are addressed on this, could be merged

**Comment by kiritigowda** [comment]
@r-abishek need merge conflicts resolved

**Comment by r-abishek** [comment]
@kiritigowda resolved

**Review by kiritigowda — APPROVED** [review]

**Review by rrawther — APPROVED** [review]

**Pull request merged**